lib/metadata/bucketfile/backend.js

```diff
@@ -15,166 +8,45 @@ import constants from '../../../constants';
 import config from '../../Config';
 
 const errors = arsenal.errors;
+const MetadataClient = arsenal.storage.metadata.client;
 
-const METADATA_PORT = 9990;
-const METADATA_PATH = `${config.filePaths.metadataPath}/`;
-const MANIFEST_JSON = 'manifest.json';
-const MANIFEST_JSON_TMP = 'manifest.json.tmp';
-const ROOT_DB = 'rootDB';
 const METASTORE = '__metastore';
 const OPTIONS = { sync: true };
```

Comment:
I think this should be on the server side to be inline with what we have in metadata now. We use ann environment variable for SYNC.